src/main.rs

```diff
@@ -18,26 +18,167 @@
 //!         ),
 //!     ]
 //! ```
+
 #![deny(missing_docs)]
 #![deny(clippy::all)]
+//#![feature(proc_macro_hygiene, decl_macro)]
 
 mod bundle;
 mod config;
 mod virtfs;
 mod workload;
 
+#[macro_use]
+extern crate serde_derive;
+
+use openssl::asn1::Asn1Time;
+use openssl::hash::MessageDigest;
+use openssl::pkey::PKey;
+use openssl::rsa::Rsa;
+use std::net::{IpAddr, SocketAddr};
+use std::path::Path;
+use warp::Filter;
+#[derive(Serialize, Deserialize)]
+struct Payload {
+    encoding: String,
+    contents: Vec<u8>,
+}
+
 use log::info;
+/// Source of the key to use for TLS
+//pub const KEY_SOURCE: &str = "file-system";
+pub const KEY_SOURCE: &str = "generate";
 
-fn main() {
-    let _ = env_logger::try_init_from_env(env_logger::Env::default());
+#[tokio::main]
+async fn main() {
+    //This required when calling from Rust std::process::command.  Recorded
+    // to allow debugging.
+    //    let args: Vec<String> = std::env::args().skip(1).collect();
+    let args: Vec<String> = std::env::args().collect();
+    let listen_address: &str = &args[0];
+    let listen_port: &str = &args[1];
```

Comment:
`keepldr` currently has no ability to pass arguments. We either need to add that or we need to hard code these addresses (for now).